components/dot_menu/dot_menu.tsx

```diff
@@ -290,6 +293,12 @@ export default class DotMenu extends React.PureComponent<Props, State> {
                 post_id: this.props.post.id,
                 root_id: this.props.post.root_id,
             },
+        }).then((res) => {
+            const callResp = (res as {data: AppCallResponse}).data;
```

Comment:
can this snippet be factored out and reused throughout somehow?

Reply:
I prefer not. Each binding should handle its own errors, and I worry the moment these two bindings start diverging we need to refactor this in one more time.